crypto/keys/types.go

```diff
@@ -44,10 +44,31 @@ type Keybase interface {
 	ExportPrivateKeyObject(name string, passphrase string) (crypto.PrivKey, error)
 }
 
+// KeyType reflects a human-readable type for key listing.
+type KeyType uint
+
+// Info KeyTypes
+const (
+	TypeLocal   KeyType = 0
+	TypeLedger  KeyType = 1
+	TypeOffline KeyType = 2
+)
+
+// String implements the stringer interface for KeyType.
+func (kt KeyType) String() string {
+	keyTypes := map[KeyType]string{
```

Comment:
This can be a constant? No need to recreate a map every time `.String()` is called.